examples/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -1418,16 +1422,14 @@
       "enum": [
         "GHI",
         "JKL"
-      ],
-      "default": "GHI"
```

Comment:
This change is what I mean. Could we ensure this doesn't go away?

Reply:
Oh I see , sorry, I will remove the code that breaks this part